### ISSUE TITLE
Don't delete newlines in one-line files

### ISSUE THREAD

**Issue by bartekpacia** [fluttercommunity/import_sorter#42]
**Describe the bug**
In my CI, I use (or at least, I would __love__ to use, because this very issue prevents me from doing so) `dart format -l 100 --output none --set-exit-if-changed .` to format my code. After that, I want to check if my imports are sorted using `flutter pub run import_sorter:main --exit-if-changed`. Now, it is just not possible for those two actions to complete successfully (they're mutually exclusive), and that is because `import_sorter` removes trailing newlines from files without any `import ...` statements (or at least it looks like so).

**File 1**
Example file that causes problems (i.e trailing newline is removed by `import_sorter`):

```dart
enum HomeEvent { showInfo, showDiscover, showProfile }
```

**File 2**
Example file that works fine (i.e trailing newline is not removed by `import_sorter`):
```dart
import 'dart:core';

enum HomeEvent { showInfo, showDiscover, showProfile }
```

**To Reproduce**
1. Create a file with contents of __file 1__
2. `dart format -l 100 .`
3. The file will have 2 lines: 1 line of code and a trailing newline. That's what I want.
3. `flutter pub run import_sorter:main --no-comments`
4. Check file contents. There won't be a trailing newline at the end of the file. I don't want this.
5. `dart format -l 100 .`. A trailing newline will be there again. Yay!

**Expected behavior**
`import_sorter` should not delete the trailing newline.

**Useful resources**
- [How to easily check if a file has a trailing newline](https://stackoverflow.com/a/47313488/7009800)
- according to my experience (I'm on macOS), the easiest way to edit a file is using TextEdit.app (it does not insert a trailing newline automatically, while several other programs I've used do insert a trailing newline (e.g `nano`)). To open a file in with TextEdit, use `open -e <file>`

**Meta Information:**
 - Dart Version: 2.12.3
 - `import_sorter` Version: 4.5.0
 - Is it a Flutter project? Yes
 - What version of Flutter are you using (if flutter project): 2.0.6

**Comment by bartekpacia** [comment]
@Matt-Gleich hey, just pinging, it's quite important for me:)

**Comment by Matt-Gleich** [comment]
Hey @bartekpacia! Sorry for the lack of communication here, I've been pretty busy with school work and some other work. I'll look into this over the weekend and let you know what's going on here.

**Comment by bartekpacia** [comment]
@Matt-Gleich I added more info to this issue (it still persists). I'd be very happy if it got fixed, but if you don't have time, I'll be done with my exams in ~2 weeks and then I can try to make a PR to fix this:)

**Comment by Matt-Gleich** [comment]
Super sorry for the lack of communication here. Just released 4.5.1 which has this fix. Thank you for the updated information and the reminder :)

**Comment by bartekpacia** [comment]
Thanks a lot! :D
